open-metadata-implementation/user-interfaces/presentation-server/README.md

```diff
@@ -54,6 +46,16 @@ the types of [Egeria OMAG Servers](../../admin-services/docs/concepts/omag-serve
   then the browser url to use to login is `https://localhost:8091/aaa`. All browser requests for this tenant will be issued using urls starting
   `https://localhost:8091/aaa`. 
     
+## FOR NEW DEVELOPMENT (as of 8/25/2020)
+ * Currently, the UI is not optimized to run in development mode. Developers must rebuild the static files every time a change is made.
+ * Because of this, work is currently underway to remake the client with Create React App. This will be done iteratively, and updates posted here.
+ * Old client code will be maintained until the cutover is complete, so developers can run the traditional way if desired.
+ * The new client is maintained in the cra-client directory.
+ * The new server is maintained in the cra-server directory.
```

Comment:
should be careful of terminology as we use 'server' to mean an egeria server, yet here I think you mean a 'process listening on a port' ie network server. We use the word platform. Not sure though that is best here. but client/server can be confusing given how much we have this all over egeria... 
(Note -- it's ok we refine the docs in a later PR - just general feedback)

Reply:
I tried to clarify a little further in the Readme, but agreed we can refine the verbiage as we go on